src/notation/internal/notationactioncontroller.cpp

```diff
@@ -892,6 +901,70 @@ void NotationActionController::addAnchoredNoteLine()
     interaction->addAnchoredLineToSelectedNotes();
 }
 
+void NotationActionController::addStretch(qreal value)
+{
+    auto interaction = currentNotationInteraction();
+    if (!interaction) {
+        return;
+    }
+
+    if (!interaction->selection()->isRange()) {
```

Comment:
Better to use currentNotationSelection()

Reply:
thanks, fixed